NotificationBox/Kwc/Component.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class NotificationBox_Kwc_Component extends Kwc_Abstract_Composite_Component
```

Comment:
Naming convention is KwcNotificationBox_Kwc_Box_Component

Reply:
I'd like an explanation for that, please